### PR TITLE
fixes and updates 

### PR DESCRIPTION
**Pull request by whokarez**
i made a couple fixes 

- disabled the collapse ability for the root of the tree 
- changed the default state of all sections to Collapsed upon initiation 
- restored the missing (Node) node and making it at the top of All nodes list 
- removed 1 item in the list that is always were called Missing nodes and had a red X for an icon 
- updated the auther credit in the plugin cfg file
- some other generic fixes here and there 
- during search the lists will auto expand if their result is inside the list , instead of staying collapsed and having to expand it manually , after the search clears they go back to their original state

### CODE REVIEW

File: class_tree.gd [hunk 1]
```diff
@@ -44,73 +44,112 @@ func generate_full_node_list() -> void:
 	# Generate the full list of node classes once
 	full_node_list.clear()  # Ensure the list is empty before populating
 	var node_classes = ClassDB.get_inheriters_from_class("Node")
+	node_classes.append("Node")  # Manually add "Node"
+
+	print("Total inheriters from Node (including 'Node'):", node_classes.size())
```
Comment (AnidemDex) [inline class_tree.gd:49]: you should consider wrapping these debug prints under certain condition

**Comment by AnidemDex** [comment]
Looks good, probably requires a rebase

**Comment by AnidemDex** [comment]
Thank you

**Comment by AnidemDex** [comment]
Please do sync commit history on your fork to match the repo history. The history log is being duplicated

File: class_tree.gd (hunk 1 above)
Comment (whokarez) [reply class_tree.gd:49]: man these prints are killing me , chat gpt keeps adding them , i was going to delete all of them but then i said no cuz they might be needed by you or someone else for debugging purposes , feel free to delete any of them if u want

**Pull request merged**